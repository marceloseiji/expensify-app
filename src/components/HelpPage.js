import React from "react";

const HelpPage = () => <div>Help page</div>;

export default HelpPage;
