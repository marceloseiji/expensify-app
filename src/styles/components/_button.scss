.button {
  border: none;
  border-radius: 2px;
  background-color: $ocean-blue;
  display: inline-block;
  color: white;
  font-weight: 300;
  line-height: 1;
  padding: $s-size $m-size;
  text-decoration: none;
}

.button--link {
  background: none;
}

.button__secondary {
  background: #888;
}