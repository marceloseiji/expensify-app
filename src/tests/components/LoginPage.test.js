import { shallow } from 'enzyme';
import React from 'react';
import { LoginPage } from '../../components/LoginPage';

test('should correctly render LoginPage', () => {
  const wrapper = shallow(<LoginPage />);
  expect(wrapper).toMatchSnapshot();
});

test('Should call startLogout on button click', () => {
  const startLogin = jest.fn();
  const wrapper = shallow(<LoginPage startLogin={startLogin} />);
  wrapper.find("button").simulate('click');
  expect(startLogin).toHaveBeenLastCalledWith();
});