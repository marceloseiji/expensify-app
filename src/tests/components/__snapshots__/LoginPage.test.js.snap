// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render LoginPage 1`] = `
<div
  className="box-layout"
>
  <div
    className="box-layout__box"
  >
    <h1
      className="box-layout__title"
    >
      Expensify
    </h1>
    <p>
      Control your expenses
    </p>
    <button
      className="button"
    >
      Login With E-mail
    </button>
  </div>
</div>
`;
